File: TimeApp/src/main/java/com/smallraw/time/broadcast/RefreshMainDataReceiver.java
package com.smallraw.time.broadcast;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public abstract class RefreshMainDataReceiver extends BroadcastReceiver {
    public static final String BroadcastRefreshMain = "LocalBroadcaseReceiver";

    public static void sendBroadcast(Context context) {
        Intent intentBroadcast = new Intent();
        intentBroadcast.setAction(RefreshMainDataReceiver.BroadcastRefreshMain);
        context.sendBroadcast(intentBroadcast);
    }
}

File: TimeApp/src/main/java/com/smallraw/time/broadcast/RefreshMainDataReceiver.kt
package com.smallraw.time.broadcast

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

open class RefreshMainDataReceiver : BroadcastReceiver() {
    companion object {
        val BroadcastRefreshMain = "LocalBroadcaseReceiver"

        fun sendBroadcast(context: Context) {
            val intentBroadcast = Intent()
            intentBroadcast.setAction(RefreshMainDataReceiver.BroadcastRefreshMain)
            context.sendBroadcast(intentBroadcast)
        }
    }

    open override fun onReceive(context: Context?, intent: Intent?): Unit {
    }

}
